contracts/native/cross_chain_manager/common/param.go

```diff
@@ -273,3 +294,35 @@ func (this *MultiSignParam) Deserialization(source *polycomm.ZeroCopySource) err
 	this.Signs = signs
 	return nil
 }
+
+type TxArgs struct {
+	ToAssetHash []byte
+	ToAddress   []byte
+	Amount      *big.Int
+}
+
+func (this *TxArgs) Serialization(sink *polycomm.ZeroCopySink) {
```

Comment:
we use zero copy or rlp, or same with evm?

Reply:
replaced with rlp serialization now